asana-helpers-markdown.js

```diff
@@ -91,7 +91,7 @@
     ));
     document.body.appendChild(buttonDiv);
 
-    waitForEditorThenConfigureIt();
+    waitForEditorThenConfigureIt().catch(err => console.error(err));
```

Comment:
without this, I was getting an unhandled error warning. The `console.error` is more useful.